mx_ide_eclipse.py

```diff
@@ -130,7 +130,7 @@ def read_core_prefs_file(self):
     class FileInfo:
         def __init__(self, path):
             self.path = path
-            with open(path) as fp:
+            with open(path, encoding='utf-8') as fp:
```

Comment:
This does not work on python2:
```
  File "/b/b/e/path with a space/mx_ide_eclipse.py", line 133, in __init__
    with open(path, encoding='utf-8') as fp:
TypeError: 'encoding' is an invalid keyword argument for this function
```
We are working to make mx require python 3.6 but are not quite there yet.